crates/types/src/logs/metadata.rs

```diff
@@ -152,6 +151,25 @@ pub enum NodeSetSelectionStrategy {
     StrictFaultTolerantGreedy,
 }
 
+impl Display for NodeSetSelectionStrategy {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::StrictFaultTolerantGreedy => write!(f, "strict-fault-tolerant-greedy"),
+        }
+    }
+}
+
+impl FromStr for NodeSetSelectionStrategy {
+    type Err = &'static str;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_str() {
+            "strict-fault-tolerant-greedy" => Ok(Self::StrictFaultTolerantGreedy),
+            _ => Err("Unknown node set selection strategy"),
+        }
+    }
+}
```

Comment:
Are we using the string representation to parse the `NodeSetSelectionStrategy` somewhere?

Reply:
it's used by `clap` in `restatectl`

Reply:
hmm, seems this is in the wrong commit. I will move it out